### PR TITLE
FRON-44 Replace Lorem Ipsum Texts and Head Description

### PR DESCRIPTION
**Pull request by MaliniakovaKarin**

### CODE REVIEW

File: src/components/Footer/Footer.vue [hunk 1]
```diff
@@ -4,16 +4,12 @@
 			class="container flex flex-col items-center sm:flex-row justify-between mx-auto text-gray-500 leading-relaxed md:leading-normal font-hairline p-6 sm:p-10"
 		>
 			<ul class="w-full sm:w-1/3 text-center md:text-left mb-4 sm:mb-0">
-				<li>Lorem ipsum</li>
-				<li>Dolor Sit Amet</li>
-				<li>Etiam dolor orci</li>
-				<li>Fermentum</li>
+				<li>Zásady ochrany osobných údajov</li>
```
Comment (MCFreddie777) [inline src/components/Footer/Footer.vue]: Please, add these into translations.

File: src/components/Footer/Footer.vue [hunk 2]
```diff
@@ -4,16 +4,12 @@
...
 			</ul>
 			<ul class="w-full sm:w-1/3 text-center md:text-left mb-4 sm:mb-0">
-				<li>Lorem ipsum</li>
-				<li>Dolor Sit Amet</li>
-				<li>Etiam dolor orci</li>
-				<li>Fermentum</li>
+				<li>Fakulta informatiky a informačných technológií</li>
```
Comment (MCFreddie777) [inline src/components/Footer/Footer.vue]: Same here.

File: src/i18n/languages/slovak.js [hunk 3]
```diff
@@ -3,12 +3,12 @@ const homePage = {
 	homePageBannerDescription: 'Po odovzdaní práce už niet cesty späť. Buď ste plagiátor alebo nie. Náš systém vám to ale môže povedať skôr abyste to dokázali včas zmeniť. Povie Vám na koľko percent máte zhodu v dokumente.',
 	homePageCallToActionButton: 'Skontrolovať moju prácu',
 	homePageGuideHeading: 'Vložte celé súbory alebo skopírujte text',
-	homePageGuideDescription: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla semper odio quis pulvinar mattis. Sed vulputate arcu.',
+	homePageGuideDescription: 'Ponúkame rýchle overenie originality vášho textu. Jednoduchým  potiahnutím súboru vložením textu si viete overiť, či vami napísaná práca je originálna a na koľko percent sa zhoduje s inými prácami. ',
```
Comment (MCFreddie777) [inline src/i18n/languages/slovak.js]: Don't forget to update english translations as well.

**Review by MCFreddie777 — CHANGES_REQUESTED** [review]
Can you please add also some description and images tags for facebook and twitter into our head description?

https://www.greengeeks.com/blog/html-tags-for-seo/#:~:text=What%20Are%20HTML%20Tags%20for,HTML%20tags%20of%20the%20site.

https://css-tricks.com/essential-meta-tags-social-media/

File: src/index.html [hunk 4]
```diff
@@ -10,20 +10,18 @@
 		<!--- TODO : <meta name="description" content="" />-->
```
Comment (MCFreddie777) [inline src/index.html]: Don't forget to uncomment and fill this one - it's really important from SEO perspective.

File: src/index.html [hunk 5]
```diff
@@ -10,20 +10,18 @@
...
-		<!-- TODO: <meta property="og:title" content="" />-->
-		<!-- TODO: <meta property="og:description" content=""/>-->
 		<meta
 			property="og:image"
 			content="<%= htmlWebpackPlugin.files.favicon %>"
 		/>
 
 		<!-- Twitter Meta Tags -->
-		<!-- TODO: <meta property="twitter:url" content="" />-->
-		<meta name="twitter:card" content="summary_large_image" />
```
Comment (MCFreddie777) [inline src/index.html:24]: Please leave this one here. It's obligatory from twitter's side:

https://css-tricks.com/essential-meta-tags-social-media/#proprietary-meta-tags

File: src/index.html [hunk 6]
```diff
@@ -10,20 +10,18 @@
 		<!--- TODO : <meta name="description" content="" />-->
 
 		<!-- Facebook Meta Tags -->
-		<!-- TODO: <meta property="og:url" content="" />-->
+		<meta property="og:title" content="Antiplag">
+		<meta property="og:description" content="Check your files before submitting.">
+		<meta property="og:url" content="https://antiplag.sk/">
```
Comment (MCFreddie777) [inline src/index.html]: What about extracting this value to the .env file?

File: src/index.html [hunk 7]
```diff
@@ -10,20 +10,18 @@
...
 		<meta property="og:type" content="website" />
-		<!-- TODO: <meta property="og:title" content="" />-->
-		<!-- TODO: <meta property="og:description" content=""/>-->
 		<meta
 			property="og:image"
 			content="<%= htmlWebpackPlugin.files.favicon %>"
 		/>
 
 		<!-- Twitter Meta Tags -->
-		<!-- TODO: <meta property="twitter:url" content="" />-->
```
Comment (MCFreddie777) [inline src/index.html:23]: Same here.

File: src/index.html [hunk 8]
```diff
@@ -10,20 +10,18 @@
 		<!--- TODO : <meta name="description" content="" />-->
 
 		<!-- Facebook Meta Tags -->
-		<!-- TODO: <meta property="og:url" content="" />-->
+		<meta property="og:title" content="Antiplag">
+		<meta property="og:description" content="Check your files before submitting.">
```
Comment (MCFreddie777) [inline src/index.html]: I'm not sure, whether this should be in slovak or english.

**Review by MCFreddie777 — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 8 above)
Comment (MaliniakovaKarin) [reply src/index.html]: I think English is OK, or should be both slovak and englis?

File: src/index.html (hunk 6 above)
Comment (MaliniakovaKarin) [reply src/index.html]: I don't understand, what I should do

File: src/index.html (hunk 8 above)
Comment (MCFreddie777) [reply src/index.html]: Not sure, I think it's a question for our PM , you better ask him.

File: src/index.html (hunk 6 above)
Comment (MCFreddie777) [reply src/index.html]: The url might differ when switching URLs. Keeping such values in configuration files (like `.env` file) is a clever idea - you can simply switch configurations without touching the code.

https://forum.vuejs.org/t/how-to-access-environment-variable-especially-inside-html-file/38862/2

File: src/index.html [hunk 9]
```diff
@@ -7,23 +7,26 @@
...
 
 		<!-- Twitter Meta Tags -->
-		<!-- TODO: <meta property="twitter:url" content="" />-->
-		<meta name="twitter:card" content="summary_large_image" />
-		<!-- TODO: <meta name="twitter:title" content="" />-->
-		<!-- TODO: <meta content="" name="twitter:description" />-->
+		<meta name="twitter:title" content="Antiplag">
+		<meta name="twitter:description" lang="sk" content="Skontroluj svoju prácu, či sa nezhoduje s prácami iných.">
+		<meta name="twitter:description" lang="en" content="Check your files before submitting.">
+		<meta name="twitter:card" content="antiplag_large_image" />
```
Comment (MCFreddie777) [inline src/index.html:25]: `<meta name="twitter:card" content="summary_large_image">` is exact value which is required by twitter. https://css-tricks.com/essential-meta-tags-social-media/#final-markup

File: src/index.html [hunk 10]
```diff
@@ -7,23 +7,27 @@
...
+		<meta property="twitter:url" content="process.env.VUE_APP_API_URL" />
```
Comment (MCFreddie777) [inline src/index.html]: Does this work? I don't see any interpolation here, if you run it in your browser, will it fill the correct value, or will it just paste `"process.env.VUE_APP_API_URL"` ?

image

File: .env.production [hunk 11]
```diff
@@ -1 +1 @@
-VUE_APP_API_URL=
\ No newline at end of file
+VUE_APP_API_URL="http://antiplag.sk"
```
Comment (MCFreddie777) [inline .env.production]: Please do not change this value, create new variable. You simply cannot change the existing variables without breaking existing functionality. 
This is the variable for API url.

**Review by MCFreddie777 — CHANGES_REQUESTED** [review]

**Review by MCFreddie777 — APPROVED** [review]

**Pull request merged**